recipes-core/rcu-sanity-check/files/post-update-sanity-check.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Service to run RCU sanity check
+After=network.target
+ConditionPathExists=/etc/rcu-mender-image-firstboot
```

Comment:
Why not check if both `/etc/rcu-mender-image-firstboot` and `/data/rcu-service/firmware/update-staged` exist before running the script? That would solve the concern about this service running during TEZI update.
Alternatively, if we want to run sanity check on first boot by default, you can check for the `/data/rcu-service/firmware/update-staged` in post-update-sanity-check script before calling mender commit/rollback.

Reply:
Good idea! I think I can add another ConditionPathExists with the pipe operator. That should work as expected.
https://stackoverflow.com/a/37313765